pygerduty/common.py

```diff
@@ -69,9 +69,28 @@ def execute_request(self, request):
         except ValueError:
             response = None
 
+        response = recurse_dict(response)
+
         return response
 
 
+def recurse_dict(d):
```

Comment:
Can you give this a more descriptive name, like `clean_response`? The current name is more "How it does it" and not "What it does"